### DISCUSSION TITLE
Anyone know why Tailwind gradients look bad on Android?

### DISCUSSION THREAD

**Discussion by jaxmerius** [ionic-team/capacitor#5389] [Q&A] [answered]
I'm using Capacitor to build an Android app from a NextJS project I have, using TailwindCSS for the styling. I'm using Tailwind's background gradient in one area, and when testing in the browser or on an android emulator on my PC the gradient looks as expected. But when I actually deploy it to my Pixel 5, the "gradient" looks more like a series of vert distinct, slightly different colored rectangles. I'm pretty new to mobile development so I'm not sure if there's an obvious fix for this or if something weird is going on; any input would be greatly appreciated!

**Comment by mlynch** [comment, marked as answer]
Got a screenshot?

**Reply by jaxmerius** [reply]
I actually just figured it out, I was using 'to-transparent' which I guess is not the proper way to make a gradient fade to transparent in Tailwind. Took that out and it works great.
